test/argentWalletDetector.js

```diff
@@ -0,0 +1,81 @@
+const ArgentWalletDetector = require("../build/ArgentWalletDetector");
```

Comment:
Branch coverage is only 50% for the `ArgentWalletDetector` contract https://1568-166830546-gh.circle-artifacts.com/0/coverage/infrastructure/ArgentWalletDetector.sol.html
Shouldn't be difficult to get it to 100% by looking at which logic branches aren't covered